### ISSUE TITLE
Tile: tile header checkbox missing text equivalent (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#953]
### Expected behavior

The checkmark has text equivalent to read out its meaning to someone who is only listening to the UI. 

### Actual behavior
The checkmark icon is visually displayed and has no text equivalent that can be read programatically.
